src/components/application_manager/test/commands/mobile/set_app_icon_test.cc

```diff
@@ -59,19 +63,27 @@ using ::utils::SharedPtr;
 using ::testing::_;
 using ::testing::Return;
 using ::testing::ReturnRef;
+using application_manager_test::MockApplicationManagerSettings;
+using protocol_handler_test::MockProtocolHandler;
+using protocol_handler_test::MockProtocolHandlerSettings;
 
 namespace {
 const uint32_t kAppId = 1u;
 const uint32_t kCmdId = 1u;
 const uint32_t kConnectionKey = 2u;
+const std::string kDirPath = "./";
+const std::string kSyncFileName = "sync_file";
 }  // namespace
 
 class SetAppIconRequestTest
     : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
+  MessageSharedPtr msg_vr;
```

Comment:
@BSolonenko Is there any reason to make this variable public?

Reply:
@AByzhynar @AKalinich-Luxoft Fixed in https://github.com/smartdevicelink/sdl_core/pull/1974/commits/beb0fd60bd310bdbc5a04821177a9005cb5c0fd9